test/groups/test_special_euclidean_group.jl

```diff
@@ -18,6 +18,7 @@ using StaticArrays
         inv,
         is_flat,
         is_identity,
+        lie_bracket,
```

Comment:
I tried to add this to enable testing, but unrelated tests seem to break now. Stil need to figure out how the testing works.

Reply:
Nice that you like the test suite, I put a reasonable amount of thinking into that.

For the proposed check the function has an `expected=` field to provide an expected value.
Checking for the types with a `isa` seems a bit too “hardcoded’ for me. But you can pass that expected value into the test suite as well.